python/src/Sample.i

```diff
@@ -786,6 +786,26 @@ Sample(PyObject * pyObj)
   return new OT::Sample( OT::convert< OT::_PySequence_, OT::Sample>(pyObj) );  
 }
 
+static Sample BuildFromSequence(PyObject * pyObj, const OT::UnsignedInteger size, const OT::UnsignedInteger dimension)
+{
+ OT::Point point( OT::convert< OT::_PySequence_, OT::Point >(pyObj) );
+ OT::UnsignedInteger pointSize = point.getDimension();
+ OT::Sample sample(size, dimension);
+ OT::UnsignedInteger k = 0;
+ for ( OT::UnsignedInteger i = 0; i < size; ++ i ) 
+ {
+   for ( OT::UnsignedInteger j = 0; j < dimension; ++ j)
+   {
+      if ( k < pointSize )
+      {
+        sample(i, j) = point[k];
+        ++ k;
+      }
+    }
+  }
+ return sample;
+}
```

Comment:
A few questions:

1) If `pointsize < size * dimension`, are additional zeros put in the sample?

2) If `pointsize > size * dimension`, then a few elements of `point` are going to be silently dropped, right?

3) Couldn't `BuildFromSequence` guess the size of the sample: `size = pointsize / dimension`? If the result is not an integer, then it could throw an exception.

Reply:
I think I will update this (we will consider that `size = pointSize`, `dimension=1`)